### PR TITLE
p1-b1-r14 Paris

### PR DESCRIPTION
**Pull request by GrantSchreib**

### CODE REVIEW

File: bHY6yh-de3795/cod-bxxqqq_bHY6yh-de3795.xml [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+              non frenaret.</quote>
+            </p>
+            <p xml:id="bHY6yh-de3795-d1e326">
+              <ref xml:id="bHY6yh-de3795-Rd1e360">Isidorus soliloquorum libro 1 <!--line jumps up--> capitulo 12</ref> <!--line jumps down--> <quote xml:id="bHY6yh-de3795-Qd1e330">Qui ve<lb ed="#B" break="no"/>niam 
+              depeccato desperat plus se 
+              <lb ed="#B"/>desperatione quam de commisso crimine 
+              <lb ed="#B"/><sic>dampernat</sic></quote>
+            </p>
+              <!--new paragraph-->¶ desperatio auget
+              <lb ed="#B"/>peccatum
```
Comment (jeffreycwitt) [inline bHY6yh-de3795/cod-bxxqqq_bHY6yh-de3795.xml:169]: In this case where there is a new entry, you're comment is great, but we also need you to create a new paragraph block.

```suggestion
            </p>
            <p>
              <!--new paragraph-->¶ desperatio auget
              <lb ed="#B"/>peccatum
            </p>
```

**Review by jeffreycwitt — CHANGES_REQUESTED** [review]
Looks pretty good. See my one comment that is preventing the xml for validating. 

Also keep working on consistent indenting within the paragraph blocks.

**Comment by GrantSchreib** [comment]
correction made

**Review by jeffreycwitt — APPROVED** [review]
Looks great. Thanks.

**Pull request merged**